packages/command-palette/src/style.scss

```diff
@@ -18,3 +22,5 @@ div.commands-command-menu {
 .commands-command-menu__header svg {
 	fill: #1e1e1e;
 }
+
+@import '~@wordpress/commands/src/style';
```

Comment:
Is this at the bottom of the file so its rules are not overridden?

Reply:
yeah. I wanted to generate the same build.css output as before, minus the color variables.